examples/svg/src/di.config.ts

```diff
@@ -48,6 +57,61 @@ export default () => {
     return container;
 };
 
+export class SVGModelSource extends LocalModelSource {
```

Comment:
If this code turns out to be useful for others, we could move it to the sprotty package.